### PR TITLE
Release 1.0.2 (cennznet/cennznet@sha256:c33d37c52a6b980ef7e93efbee738…

### PR DESCRIPTION
**Pull request by KarishmaBothara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 [Development guide](docs/GET_STARTED.md)
 
 ## Release notes
+[15/05/2020] 1.0.1
```
Comment (benxgao) [inline README.md:10]: 1.0.2?
As what I learnt from last release, this README.md may not reflect on npm. The actual place may need to change is `cennznet/types/README.md`?

**Comment by benxgao** [comment]
Is it updated via script like `yarn run lerna version 1.0.2 --no-git-tag-version --no-push`, if so LGTM.

File: README.md (hunk 1 above)
Comment (KarishmaBothara) [reply README.md:10]: I feel we might not want to put release notes in README at all..they should not be part of read me

**Comment by KarishmaBothara** [comment]
> Is it updated via script like `yarn run lerna version 1.0.2 --no-git-tag-version --no-push`, if so LGTM.

yes

File: packages/api/test/e2e/tx.e2e.ts [hunk 2]
```diff
@@ -32,6 +33,23 @@ describe('e2e transactions', () => {
...
+    //         {
+    //           initialIssuance: 0,
+    //           permissions: {
+    //             update: null,
+    //             mint: null,
+    //             burn: null,
+    //           },
+    //         })
+    //     ))
+    //   .signAndSend(sudoPair);
```
Comment (alexsednz) [inline packages/api/test/e2e/tx.e2e.ts:52]: Leaving commented dead code is not good.

File: README.md (hunk 1 above)
Comment (benxgao) [reply README.md]: Yeah, agreed. It is better in `CHANGELOG.md`. As the time we refactored on API, we did not have considered a proper CHANGELOG at all, so decided to put a simple release notes at the top of README.

Could consider move to a changelog, and make a link to readme, or anyway consumers can easily find this kind of release notes..

**Pull request merged**